### PR TITLE
[doc](insert) Add group commit docs

### PR DESCRIPTION
**Pull request by mymeiyi**
## Proposed changes

Add doc for group commit.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/en/docs/data-operate/import/import-way/group-commit-manual.md [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+-->
+
+# Group Commit
+
+Group commit load does not introduce a new import method, but an extension of `INSERT INTO tbl VALUS(...)`、`Stream Load`、`Http Stream`.
+
+In Doris, all methods of data loading are independent jobs which initiate a new transaction and generate a new data version. In the scenario of high-frequency writes, both transactions and compactions are under great pressure. Group commit load reduces the number of transactions and compactions by combining multiple small load tasks into one load job, and thus improve write performance.
+
+It should be noted that the group commit is returned after the data is writed to WAL, at this time, the data is not visible for users, the default time interval is 10 seconds.
+
```
Comment (dataroaring) [inline docs/en/docs/data-operate/import/import-way/group-commit-manual.md:34]: after a fixed time interval, the data is committed, then the data is visible for users, the default time interval is 10 seconds.

**Review by dataroaring — CHANGES_REQUESTED** [review]

**Review by dataroaring — DISMISSED** [review]
LGTM

File: docs/zh-CN/docs/admin-manual/config/be-config.md [hunk 2]
```diff
@@ -1534,3 +1534,16 @@ load tablets from header failed, failed tablets size: xxx, path=xxx
 
 * 描述: BE 是否开启使用java-jni，开启后允许 c++ 与 java 之间的相互调用。目前已经支持hudi、java-udf、jdbc、max-compute、paimon、preload、avro
 * 默认值: true
+
+#### `group_commit_interval_ms`
+
+* 描述:  攒批写入开启多久后结束，单位为毫秒，请参考 [Group Commit](../../data-operate/import/import-way/group-commit-manual.md)
```
Comment (hust-hhb) [inline docs/zh-CN/docs/admin-manual/config/be-config.md:1540]: 攒批提交间隔，攒批开启后多久会执行一次提交

File: docs/zh-CN/docs/data-operate/import/import-way/group-commit-manual.md [hunk 3]
```diff
@@ -0,0 +1,416 @@
...
+
+攒批写入有三种模式，分别是：
+
+* `off_mode`
+
+不开启攒批，保持以上三种导入方式的默认行为。
+
+* `sync_mode`
+
+多个用户导入复用一个内部导入，等内部导入成功或失败后，外部导入才会返回。
```
Comment (hust-hhb) [inline docs/zh-CN/docs/data-operate/import/import-way/group-commit-manual.md:43]: 这里的外部导入改成客户端会不会好一点，用户可能不理解外部导入的含义

File: docs/zh-CN/docs/data-operate/import/import-way/group-commit-manual.md [hunk 4]
```diff
@@ -0,0 +1,416 @@
...
+
+如果内部导入成功，数据可以立即查出。
+
+* `async_mode`
+
+多个用户导入复用一个内部导入，内部导入将处理后的数据写入Write Ahead Log(WAL)后，立即返回。
+
+此时，数据不能立即读出。等内部导入完成后，数据才能读出。
```
Comment (hust-hhb) [inline docs/zh-CN/docs/data-operate/import/import-way/group-commit-manual.md:51]: 这里要不要加上延迟可见时间的大概值

File: docs/zh-CN/docs/data-operate/import/import-way/group-commit-manual.md [hunk 5]
```diff
@@ -0,0 +1,416 @@
...
+  * 但攒批模式支持了一定程度的`max_filter_ratio`语义，当导入的总行数不高于`group_commit_memory_rows_for_max_filter_ratio`(配置在be.conf中，默认为10000行)，会把数据缓存起来，计算出真正的`filter_ratio`，如果超过了`max_filter_ratio`，会把数据丢弃，用户导入失败
+
+
+* WAL限制
+
+  * 对于`async_mode`的攒批写入，会把数据写入WAL。如果内部写入成功，则WAL被立刻删除；如果内部导入失败，通过导入WAL的方法来恢复数据
+
+  * 目前WAL文件只存储在一个BE上，如果这个BE磁盘损坏或文件误删等，可能导入丢失部分数据。
+
+  * 当下线BE节点时，请使用[`DECOMMISSION`](../../../sql-manual/sql-reference/Cluster-Management-Statements/ALTER-SYSTEM-DECOMMISSION-BACKEND.md)命令，安全下线节点，防止该节点的WAL文件还没有全部处理完成，导入部分数据丢失
```
Comment (hust-hhb) [inline docs/zh-CN/docs/data-operate/import/import-way/group-commit-manual.md]: 安全下线节点，防止该节点下线前WAL文件还没有全部处理完成，导入部分数据丢失

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by Yukang-Lian — APPROVED** [review]

**Comment by dataroaring** [comment]
run buildall

**Pull request merged**